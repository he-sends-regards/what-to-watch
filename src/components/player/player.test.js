import React from "react";
import renderer from "react-test-renderer";
import Player from "./player";

import mockFilms from "../../mocks/films";

it(`Player snapshot test`, () => {
  const tree = renderer
    .create(<Player films={mockFilms} filmId={1} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
