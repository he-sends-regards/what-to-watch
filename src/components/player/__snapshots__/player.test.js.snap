// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player snapshot test 1`] = `
<div
  className="player"
>
  <video
    autoPlay={true}
    className="player__video"
    controls={true}
    src="http://media.xiph.org/mango/tears_of_steel_1080p.webm"
  />
  <div
    id="countdown"
  >
    Video ends after 
    <span
      id="countdown"
    >
      xx
    </span>
     seconds.
  </div>
  <button
    className="player__exit"
    onClick={[Function]}
    type="button"
  >
    Exit
  </button>
</div>
`;
